webpack/scenes/Subscriptions/UpstreamSubscriptions/UpstreamSubscriptionsPage.js

```diff
@@ -11,6 +11,7 @@ import { urlBuilder } from 'foremanReact/common/urlHelpers';
 import { LoadingState } from '../../../components/LoadingState';
 import { Table } from '../../../components/pf3Table';
 import { columns } from './UpstreamSubscriptionsTableSchema';
+import api, { orgId } from '../../../services/api';
```

Comment:
These imports don't seem to be used anywhere in this file. I know we saw them yesterday, so not sure what happened?

Reply:
Good catch. I ended up removing the subscription switcher since I didn't believe it made sense in this context. That orphaned these imports.

Reply:
gotcha, makes sense! :+1: